seed/serializers/certification.py

```diff
@@ -0,0 +1,360 @@
+                    ) for url in urls
+                ]
+            )
+            instance.log(
+                user=user,
+                changed_fields="added urls: {}".format(urls),
+                description="Urls added by update serializer method",
+                parent=log,
+                ancestor=log.ancestor
+            )
+            if deleted:
+                instance.log(
+                    user=user,
+                    changed_fields="deleted urls: {}".format(urls),
+                    description="Urls deleted by update serializer method",
+                    parent=log,
+                    ancestor=log.ancestor
+                )
+        return instance
+
+    def validate(self, data):
+        """Object level validation."""
+        # Only rating, metric can be supplied
+        scores = ([
+            elem for elem in [
+                data.get('metric'), data.get('rating')
+            ] if elem is not None
+        ])
+        if len(scores) > 1:
+            msg = 'Only one of metric or rating can be supplied.'
+            raise ValidationError(msg)
+
+        # check rating/metric type
+        assessment = data.get('assessment')
+        if not assessment and self.instance:
+            assessment = getattr(self.instance, 'assessment', None)
+        if not assessment:
+            raise ValidationError('Could not find assessment.')
+
+        rating = data.get('rating')
+        if rating and assessment.is_numeric_score:
+            msg = "{} uses a metric (numeric score).".format(assessment.name)
+            raise ValidationError(msg)
+        elif rating and not isinstance(rating, basestring):
+            raise ValidationError('Rating must be a string.')
+
+        metric = data.get('metric')
+        if metric and not assessment.is_numeric_score:
+            msg = "{} uses a rating (non numeric score).".format(
+                assessment.name
+            )
+            raise ValidationError(msg)
+        if metric:
+            try:
+                float(metric)
+            except ValueError:
+                raise ValidationError('Metric must be a number.')
+            if assessment.is_integer_score and not float(int(metric)) == metric:
+                raise ValidationError('Metric must be an integer.')
+        # validate org_ids match
+        return super(GreenAssessmentPropertySerializer, self).validate(data)
+
+
+class GreenAssessmentURLSerializer(OrgValidateMixin, serializers.ModelSerializer):
+    # ensure reuqest.users org matches that property_assessment
+    org_validators = [ASSESSMENT_PROPERTY_VALIDATOR]
+
+    class Meta:
+        model = GreenAssessmentURL
+
+
+class GreenAssessmentPropertyReadOnlySerializer(serializers.BaseSerializer):
+    """Simple read only Serializer describing Green Assessment attached to
+    property. Use with prefetch_related to avoid extra database calls.
+    """
+
+    def to_representation(self, obj):
```

Comment:
Using to_representation is the correct method for customizing the serialization of a serializer class 
http://www.django-rest-framework.org/api-guide/serializers/#overriding-serialization-and-deserialization-behavior